external/sentry.go

```diff
@@ -0,0 +1,31 @@
+package external
+
+import (
+	"github.com/getsentry/sentry-go"
+	log "github.com/sirupsen/logrus"
+	"golbat/config"
+	"time"
+)
+
+const sentryDsn = "https://bc0b5dc992ee486a90faa20b83016351@sentry.mew.is/2"
+
+func InitSentry() {
+	if !config.Config.Sentry.Disable {
+		log.Infof("Sentry init")
+
+		targetDsn := sentryDsn
+		if config.Config.Sentry.DSN != "" {
+			targetDsn = config.Config.Sentry.DSN
+		}
+
+		err := sentry.Init(sentry.ClientOptions{
+			Dsn:        targetDsn,
+			Debug:      false,
+			SampleRate: config.Config.Sentry.SampleRate,
+		})
+		if err != nil {
+			log.Errorf("Sentry Init Failed: %s", err)
+		}
+		defer sentry.Flush(2 * time.Second)
```

Comment:
this won't defer anything (it will execute immediately) - intended?

Reply:
You are correct, I'll move it 👍

Reply:
I am not sure what it is meant to be doing - ie when Flush needs to be called